common/workunit/workunit.cpp

```diff
@@ -9492,9 +9492,25 @@ IPropertyTree *CLocalWorkUnit::getUnpackedTree(bool includeProgress) const
     return ret.getClear();
 }
 
+IPropertyTree *CLocalWorkUnit::queryIndirectTree() const
```

Comment:
long term I think you will want to access a single graph at a time, rather than all of them.  Probably sufficient for now though.